server/src/main/scala/de/frosner/broccoli/nomad/NomadModule.scala

```diff
@@ -37,5 +38,5 @@ class NomadModule extends AbstractModule with ScalaModule {
   @Provides
   @Singleton
   def provideNomadClient(config: NomadConfiguration, wsClient: WSClient): NomadClient =
-    new NomadHttpClient(config.url, wsClient)(play.api.libs.concurrent.Execution.defaultContext)
+    new NomadHttpClient(Uri.parse(config.url), wsClient)(play.api.libs.concurrent.Execution.defaultContext)
```

Comment:
IOW, I didn't make this change to address #81, but just to build URLs in the client in a simpler way, and I don't think that it'll result in better error messages.

To address #81 you'd need to validate the URL upfront while parsing the configuration, to be able to throw an error message that can still refer to the path of the malformed value.